src/pages/challenges.tsx

```diff
@@ -29,14 +30,14 @@ const renderCardList = (
 
 const Challenges: FC = () => {
     const [filteredChallenge, setFilteredChallenge] = useState<any>([]);
-    const { data: user } = useGetUserQuery(skipToken);
+    const { data: user, isLoading: isAuthenticating } =
```

Comment:
raising this concern about having meaningful variable naming although as confirmed by @bhtibrewal there is already an issue raised to fix the changes for `isAuthenticating ` and  `isLoggedIn`  which seem some what same to me.

Also please resolve this conversation if needs to be addressed later

Reply:
it is being addressed